src/App.js

```diff
@@ -1,64 +1,63 @@
-import React, { Component } from 'react';
+/*
+   Edited by: Maria Babko
+   Date March 2021
+
+   Renders a Todolist page.
+*/
+import React, { useState } from 'react';
+
+import Alert from 'react-bootstrap/Alert';
+import ManageTodolist from './components/ManageTodolist';
 
 /**
- * App class runs entire application
- */
-class App extends Component {
+  * Sends POST request to TodolistAPI to add todo-list item.
+  */
+const App = () => {
+   const [alertStatus, setAlertStatus] = useState(false);
+   const [currentAlert, setAlert] = useState('');
+   const [alertType, setAlertType] = useState('');
+
+
    /**
-   * Represents a book.
-   * @constructor
-   */
-   constructor() {
-      super();
-   }
+    * Creates a new alert and displays it to the user
+    * After 2 seconds, clears the alarm.
+    * @param {String} message - The alert message to display
+    */
+   const createAlert = (message, type) => {
+      setAlertType(type);
+      setAlert(message);
+      setAlertStatus(true);
 
-   render() {
-      return (
-      <div className="App">
-         <button className="btn btn-primary">Add Item</button>
-         <ul>
-            <li>
-               <div class="card" style={{width: '18rem'}}>
-                  <div class="card-body">
-                     <h5 class="card-title">Go Shopping</h5>
-                     <h6 class="card-subtitle mb-2 text-muted">January 5, 2021</h6>
-                     <p class="card-text">
-                        Some quick example text to build on the card title and make up the bulk of
-                        the card's content.
-                     </p>
-                  </div>
-               </div>
-            </li>
+      setTimeout(() => {
+         setAlert('');
+         setAlertStatus('');
+         setAlertStatus(false);
+      }, 2000);
+   }
```

Comment:
-0 Missing semi-colon.

While JS inserts semi-colons for you, it should not be relied upon and it is a best practice to always include them yourself.